packages/login/network/settings.js

```diff
@@ -347,3 +380,55 @@ export const isTestingNetwork = (networkId, providerId) => {
   const net = getNetworkById(networkId, providerId)
   return net.id !== NETWORK_MAIN_ID
 }
+
+export const networkSelectorGroups = [
+  {
+    title: 'Production networks',
+    description: 'Manage your funds',
+    providers: [
+      {
+        provider: providerMap.chronoBank,
+        network: chronoBankMainnet,
+      },
+      {
+        provider: providerMap.infura,
+        network: infuraMainnet,
+      },
+      {
+        provider: providerMap.mew,
+        network: mewMainnet,
+      },
+      {
+        provider: providerMap.giveth,
+        network: givethMainnet,
+      },
+    ],
+  },
+  {
+    title: 'Test networks',
+    description: 'Test networks with fake funds',
+    providers: [
+      {
+        provider: providerMap.chronoBank,
+        network: chronoBankTestnet,
+      },
+      {
+        provider: providerMap.infura,
+        network: infuraTestnet,
+      },
+    ],
+  },
+  {
+    title: 'Developer networks',
+    providers: [
+      {
+        provider: providerMap.local,
+        network: infuraLocalNetwork,
+      },
+      process.env.NODE_ENV === 'development' ? {
```

Comment:
Could you please replace `process.env.NODE_ENV` to `process.env['NODE_ENV']`
This is a magic for ChronoMint mobile project.